app/javascript/app/components/country-ndc-sdg-linkages/country-ndc-sdg-linkages-component.jsx

```diff
@@ -55,20 +55,53 @@ class CountrySDGLinkages extends PureComponent {
     ) : null;
   }
 
-  render() {
+  renderCards() {
```

Comment:
I don't understand the reason for this function? It does that same as below with a different name. I dont mind too much but can we not call it render'something' as it is confusing with react render. We have been using getSomething elsewhere in the app I believe.